.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true
+  },
+  extends: ["airbnb-base"],
+  globals: {
+    Atomics: "readonly",
+    SharedArrayBuffer: "readonly"
```

Comment:
What do these do?

Reply:
airbnb is a styleguide picked from the CLI setup, I don't know what the globals do. They're probably just from the setup.